src/data_import/json_reader.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+from data_import import FileReader
+import pandas as pd
+
+
+class JSONReader(FileReader):
+    def __init__(self, filepath: str):
+        self.error = ''
+        try:
+            super().__init__(filepath)
+            self.columns_name = list(pd.read_json(self.filepath).columns)
+        except FileNotFoundError:
+            self.error = 'This filepath: {} is invalid. Please write correct path.'.format(filepath)
+        except Exception:
+            self.error = 'There is some problem with file. Please try again.'
+
+        # if file is big we can not read by chunks because of .json format
+        if self.need_chunks:
+            self.error = 'File is to big for parsing in .json format'
+        self.reader = None
```

Comment:
```suggestion
    def __init__(self, filepath: str):
        self.error = ''
        try:
            super().__init__(filepath)
            self.columns_name = list(pd.read_json(self.filepath).columns)
            if self.need_chunks:
                self.error = 'File is to big for parsing in .json format'
            self.reader = None
        except FileNotFoundError:
            self.error = 'This filepath: {} is invalid. Please write correct path.'.format(filepath)
        except Exception:
            self.error = 'There is some problem with file. Please try again.'
```

Bug with filepath when need_chunks wants to use non-existing filepath. I suggest simple change, maybe in read csv class should be done similary?

Reply:
In csv format we can read data by chunks

Reply:
Oh, my mistake. Absolutely you're right